bin/test

```diff
@@ -5,5 +5,8 @@
 # See: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin
 set -e
 
+npm install maildev -g
```

Comment:
It feels odd to me that we're doing the installation as part of the test script; since it's installing a package it feels like it wants to live in `bin/setup`? Also, I believe we encourage folks to run `bin/setup` every time they begin work... But I could be wrong here...